### PR TITLE
Question service/scraper

### LINKED ISSUE #30: Scrape Leetcode
Scrape leetcode website to get the necessary data (title, description, type (array, dynamic programming), etc)

### PR DESCRIPTION
**Pull request by drake25122000**
Resolves #30 #37

### CODE REVIEW

File: question-service/scraper/tracker.conf [hunk 1]
```diff
@@ -0,0 +1 @@
+9
```
Comment (erinmayg) [inline question-service/scraper/tracker.conf:1]: no line at EOF

File: question-service/scraper/main.py [hunk 2]
```diff
@@ -0,0 +1,210 @@
...
+    with open(TRACKER_FILENAME, "r") as f:
+        return int(f.readline())
+    
+completed_upto = read_tracker()
+
+def update_tracker(num):
+    with open(TRACKER_FILENAME, "w") as f:
+        f.write(str(num))
+
+
```
Comment (erinmayg) [inline question-service/scraper/main.py:45]: extra blank lines

File: question-service/scraper/main.py [hunk 3]
```diff
@@ -0,0 +1,210 @@
...
+# Setup Selenium Webdriver
+CHROMEDRIVER_PATH = r"./driver/chromedriver.exe"
+options = Options()
+options.headless = True
+# Disable Warning, Error and Info logs
+# Show only fatal errors
+options.add_argument("--log-level=3")
+driver = webdriver.Chrome(executable_path=CHROMEDRIVER_PATH, options=options)
+
+
```
Comment (erinmayg) [inline question-service/scraper/main.py:59]: extra blank lines

**Review by erinmayg — COMMENTED** [review]
Minor code style changes

**Review by jzwoo — APPROVED** [review]
LGTM

**Review by florenciamartina — APPROVED** [review]
LGTM

**Pull request merged**